TTS/elevenlabs.py

```diff
@@ -0,0 +1,46 @@
+import random
+
+from elevenlabs import generate, save
+
+from utils import settings
+
+
+class elevenlabs:
+    def __init__(self):
+        self.max_chars = 5000
+        self.voices = ["Adam", "Antoni", "Arnold", "Bella", "Domi", "Elli", "Josh", "Rachel", "Sam"]
+
+    def run(
+        self,
+        text: str,
+        filepath: str,
+        random_voice=False,
+    ):
+        voice_name = settings.config["settings"]["tts"]["elevenlabs_voice_name"]
+        if voice_name == "":
+            voice_name = "Bella"
+            raise ValueError(
+                "set elevenlabs name value to a valid value, switching to default voice (Bella)"
+            )
```

Comment:
```suggestion
            print(
                "You didn't set an Elevenlabs name value to a valid value, switching to default voice (Bella)"
            )
```
Btw this statement will never be reached since it's required in the config